runtime-library/README.md

```diff
@@ -64,9 +68,21 @@ cmake -S . -B build-ios-sim -GNinja \
   -DCMAKE_INSTALL_PREFIX=../build-ios-sim/install \
   -DIREE_BUILD_COMPILER=OFF
   
-cmake --build build
+cmake --build build-ios-sim
 ```
 
-This will give us the test binary `build-ios-sim/bin/ireert_test.app` and the IREE runtime library `build-ios-sim/lib/libireert.a`.
+This will give us the app bundle `build-ios-sim/bin/ireert_test.app` and the IREE runtime framework `build-ios-sim/lib/ireert.framework`.
 
 To configure and build for iOS devices, we can change `-DCMAKE_OSX_SYSROOT=$(xcodebuild -version -sdk iphonesimulator Path)` into `-DCMAKE_OSX_SYSROOT=$(xcodebuild -version -sdk iphoneos Path)`.
+
+## Build Apple XCFramework
+
+An [XCFramework
+bundle](https://developer.apple.com/documentation/xcode/creating-a-multi-platform-binary-framework-bundle)
+is a binary package created by Xcode that includes the frameworks and
+libraries necessary to build for multiple platforms (iOS, macOS, tvOS,
+and watchOS), including Simulator builds.  Run the following command to build the `ireert.xcframework`.
+
+```
+./create_xcframework.sh
```

Comment:
Note in a comment the expected directory layout? I see that the script is hard-coded to find the CMake build directory relative to the iree-samples repo. We may want to rework this at some point, so fine just noting it for now.

Reply:
I agree. I do not have a good idea how at the moment.  To get the script, users need to check out some GitHub repo.  Maybe the IREE runtime as a standalone repo, so we could add IREE as a git submodule.

How about we keep the script for now as a record of what we explored, but remove the description from README.md?